Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/messaging/service/decorator/HttpHeaderServiceEndpointDecoratorTest.java

```diff
@@ -69,78 +71,104 @@ public void tearDown() {
+
     @Test
     public void testConfigure_WithPropertyValues() throws PropertyAccessException {
-        CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(
-                new TestServicePortDescriptor());
+        final CONNECTTestClient<TestServicePortType> testClient = new CONNECTTestClient<>(
+            new TestServicePortDescriptor());
+
+        final ServiceEndpoint<TestServicePortType> serviceEndpoint = testClient.getServiceEndpoint();
 
-        ServiceEndpoint<TestServicePortType> serviceEndpoint = testClient.getServiceEndpoint();
-        
         final PropertyAccessor mockPropAccessor = mock(PropertyAccessor.class);
 
-        AssertionType assertion = new AssertionType();
-        
-        HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(serviceEndpoint, assertion) {
-            
+        final AssertionType assertion = new AssertionType();
+
+        final HttpHeaderServiceEndpointDecorator headerDecorator = new HttpHeaderServiceEndpointDecorator(
+            serviceEndpoint, assertion) {
+
             @Override
             protected PropertyAccessor getPropertyAccessor() {
                 return mockPropAccessor;
             }
         };
-        
+
         final String customHeaderName = "customName";
         final String customHeaderValue = "customValue";
-        Set<String> headerNames = new HashSet<>();
+        final Set<String> headerNames = new HashSet<>();
         headerNames.add(NhincConstants.CUSTOM_HTTP_HEADERS + "." + customHeaderName);
-        
-        when(mockPropAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.KEEP_ALIVE_PROP)).thenReturn("True");
+
+        when(mockPropAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.KEEP_ALIVE_PROP))
+            .thenReturn("True");
         when(mockPropAccessor.getPropertyNames(NhincConstants.GATEWAY_PROPERTY_FILE)).thenReturn(headerNames);
-        when(mockPropAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE, NhincConstants.CUSTOM_HTTP_HEADERS + "." + customHeaderName)).thenReturn(customHeaderValue);
-        
+        when(mockPropAccessor.getProperty(NhincConstants.GATEWAY_PROPERTY_FILE,
+            NhincConstants.CUSTOM_HTTP_HEADERS + "." + customHeaderName)).thenReturn(customHeaderValue);
+
         headerDecorator.configure();
-        
+
         validateConfiguration(headerDecorator);
-        BindingProvider bp = (BindingProvider) headerDecorator.getPort();
-        Map<String, List<String>> bpMap = (Map<String, List<String>>) bp.getRequestContext().get(NhincConstants.CUSTOM_HTTP_HEADERS);
-        assert(bpMap.containsKey("customName"));
+        final BindingProvider bp = (BindingProvider) headerDecorator.getPort();
+        final Map<String, List<String>> bpMap = (Map<String, List<String>>) bp.getRequestContext()
+            .get(NhincConstants.CUSTOM_HTTP_HEADERS);
+        assert bpMap.containsKey("customName");
```

Comment:
I think you forget this line.  remove assert  and start to use junit library.  Recommend to use assertTrue(bpMap.containsKey("customName"));